config/i18n.json

```diff
@@ -826,6 +826,9 @@
     "Unknown": "Challenge",
     "Weekly": "Weekly Challenge"
   },
+  "Mods": {
+    "HarmonicModDescription": "This mod's effect comes at a reduced cost and changes depending on the subclass used."
```

Comment:
```suggestion
    "HarmonicModDescription": "This mod's effect comes at a reduced cost and changes element depending on the equipped subclass."
```

out of curiosity is this string from somewhere? it seems familiar. a game tooltip? entirely written by you? it's pretty good.

Reply:
This is not an in-game message but one that I wrote out to depict what the mod does. The mod has 5 different states and the description of the mod changes depending on it the bungie api exposes all 5 different descriptions but none of them are general descriptions that depict what the mod does. see below for examples

![image](https://github.com/DestinyItemManager/DIM/assets/98132549/9386ca57-e5a5-4425-bfd1-3cab4448e069)
![image](https://github.com/DestinyItemManager/DIM/assets/98132549/c10ed4db-18e8-4cb7-98b7-21657008ec04)